### PR TITLE
Comment discussion limit size

### PR DESCRIPTION
**Pull request by rich-nguyen**
Adds functionality that limits the size of discussions that can work with the 'all' option.
screen shot 2015-02-03 at 12 43 35

### CODE REVIEW

File: discussion/app/views/discussionComments/commentsList.scala.html [hunk 1]
```diff
@@ -15,3 +18,11 @@
 
     @fragments.reportComment()
 </div>
+
+@userMessageForLargeDiscussion = {
+    @if(page.isLargeDiscussion){
+        <div class="d-discussion__size-message">
+            Due to the large number, comments are being shown 100 per page.
```
Comment (mattosborn) [inline discussion/app/views/discussionComments/commentsList.scala.html]: Due to the large number of comments, page size is limited to 100.

??

**Comment by mattosborn** [comment]
:+1: for code review but :-1: for the feature. Sorry to keep saying this but I still massively disagree with how we are implementing this for a few reasons and IMO we should remove it. We should be spending our time and energy elsewhere.

File: discussion/app/views/discussionComments/commentsList.scala.html (hunk 1 above)
Comment (rich-nguyen) [inline discussion/app/views/discussionComments/commentsList.scala.html]: I'm apathetic towards the wording. @cantlin ?

**Comment by rich-nguyen** [comment]
agreed!

**Pull request merged**